docs/advanced_examples/ConvolutionalNeuralNetwork.ipynb

```diff
@@ -688,13 +515,9 @@
    "source": [
     "### Conclusion\n",
     "\n",
-    "We see that quantization with **3** bit weight and activations is the best viable FHE configuration,\n",
-    "as the accumulator bit width for this configuration is between **7 and 8** bits (can vary due to the final \n",
-    "distribution of the weights). The accuracy in this setting, 92% is a few percentage points \n",
-    "under the maximum accuracy achievable with larger accumulator bit widths (97-98%). \n",
+    "In this example, a simple CNN model is trained with torch and reach 98% accuracy in clear. The model is then converted to FHE and evaluated over 100 samples in FHE.\n",
```

Comment:
99 right ?

Reply:
The torch model over the entire test set is 98... and the model in fhe over a subset from the test of 100 samples is 99%. I did not mention the accuracy but just say it's similar. Not sure what's best.

Reply:
ok fine I guess